.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+### Solves Issue #<number>
```

Comment:
This line is to be mentioned in description itself and not in title <--- Mention this

Reply:
Just to make sure we're on the same page.
```
### Solves Issue #<number>

Description: <provide a description> 
Title should be small and relevant.
```